sound/soc/intel/common/soc-acpi-intel-cml-match.c

```diff
@@ -9,45 +9,59 @@
 #include <sound/soc-acpi.h>
 #include <sound/soc-acpi-intel-match.h>
 
-static struct snd_soc_acpi_codecs cml_codecs = {
+static struct snd_soc_acpi_codecs rt5682_codecs = {
 	.num_codecs = 1,
 	.codecs = {"10EC5682"}
 };
 
-static struct snd_soc_acpi_codecs cml_spk_codecs = {
-	.num_codecs = 1,
-	.codecs = {"MX98357A"}
+static struct snd_soc_acpi_codecs rt5682_spk_codecs = {
+	.num_codecs = 2,
+	.codecs = {"10EC5682", "MX98357A"}
+};
+
+static struct snd_soc_acpi_codecs da7219_spk_codecs = {
+	.num_codecs = 2,
+	.codecs = {"10EC5682", "MX98357A"}
```

Comment:
@aiChaoSONG where is the da7219 is the codecs list?

Reply:
Oh, forgive me, da7219 is the codecs list, and the '10EC5682' here should be 'DLGS7219'. will fix this in next push